rts/Game/Camera/OverviewController.cpp

```diff
@@ -4,18 +4,25 @@
 #include "OverviewController.h"
 
 #include "Map/Ground.h"
-#include "Map/ReadMap.h"
 #include "Game/UI/MiniMap.h"
 #include "Game/UI/MouseHandler.h"
-#include "Sim/Misc/GlobalConstants.h"
 #include "System/Log/ILog.h"
 
+static float fit_cam_height_to_map(float map_x, float map_y, float fov) {
+    const auto extra_margin = 1.037;
```

Comment:
`static constexpr float EXTRA_MARGIN = 1.037f;`

Also does this number represent anything? How was it derived? Please leave some comment even if it's been selected empirically.

Reply:
I carried it over from the previous implementation so users won't notice the change. It was written as 2.5 for 45 FOV which is 1.037 bigger than the actual field of view.